### ISSUE TITLE
In kodi 16.1 and plexmbc 4.0 beta4 - "recently added" movies doesnt show by date.

### ISSUE THREAD

**Issue by saima2** [HooliganHarls/plugin.video.plexbmc#7]
Using Kodi 16.1 on nvidea shield box.

using plexmbc 4.0 beta 4.

When i go to movies and browse by "recently added" it doesnt show it by the latest movies added in stead it just shows in alphabetical order.

any way to fix this? cuz i have so many movies, and its not fun when I cant see the latest movies on top once I enter that section.

**Comment by d0cx** [comment]
This issue also happens to me on LibreELEC v7.0.1 + Kodi Jarvis 16.1, using hippojay's 4.1.0 isengard plugin. It seems to be that Jarvis removed the "Default" sorting option on the file lists.

**Comment by d0cx** [comment]
Please check the ticket for the fix. A pull request that has not yet been merged has the fix. I manually changed the lines and it worked!

**Comment by d0cx** [comment]
https://github.com/hippojay/plugin.video.plexbmc/issues/172

**Comment by hippojay** [comment]
yeah - sorry.  Jarvis is something i'm going to be looking at soon, as I
have updated my setup too with Openelec 7 beta.  I'll check out the pull
request this weekend when I get my media server back on line (moving
servers)..

On 17 June 2016 at 01:02, d0cx notifications@github.com wrote:

> hippojay#172 https://github.com/hippojay/plugin.video.plexbmc/issues/172
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/HooliganHarls/plugin.video.plexbmc/issues/7#issuecomment-226647728,
> or mute the thread
> https://github.com/notifications/unsubscribe/AAVvedtyVEA_q3U91Z4zlaFqKSBEH4m1ks5qMeQkgaJpZM4Ij_cR
> .

## 

Dave

**Comment by molle83** [comment]
Hi!
I'am new with raspberry pi and plexbmc.
I newly installed plexbmc 4.0 beta 4 on my raspberry pi running kodi 16 with xbian and i also have problems with sorting with "recently added"
I found a solution to solve the problem here https://github.com/hippojay/plugin.video.plexbmc/pull/160/files but for me it didnt work, proberly i have done something wrong when changed the lines in the py-files "plexsignin.py" and plexbmc.py.

[plexbmc.txt](https://github.com/HooliganHarls/plugin.video.plexbmc/files/402470/plexbmc.txt)
[plex_signin.txt](https://github.com/HooliganHarls/plugin.video.plexbmc/files/402469/plex_signin.txt)

I found this files in \XBIAN\xbian-kodi\addons\plugin.video.plexbmc-4.0.0beta4\resources\lib and not in a "plex"-folder.
Another thing is that i dont the folder "lib" with the file named "**init**.py" in resources/lib/ndg/.

Is thete a new version of plexbmc where this problem is resolved or can some one help me with the py-files?
